### PR TITLE
Include fzf.fish

### PR DESCRIPTION
**Pull request by PatrickF1**
You might be wondering, what about Jethro Kuan's famous https://github.com/jethrokuan/fzf plugin?
I used it for years and to be sure it's a great plugin. I just eventually realized the keybindings didn't make sense and the it packed way too much power/flexibility to be easily maintained or configured, so I wrote my own. My plugin also includes an additional feature of searching through shell variables.

### CODE REVIEW

**Comment by derekstavis** [comment]
It's a bit confusing to have two FZF plugins but I guess people will choose the one they prefer. @oh-my-fish/core thoughts?

**Comment by patrickf3139** [comment]
@derekstavis if you think it helps, I could mention in the description that this is a simpler, lighterweight alternative to Jethro Kuan's version.

**Comment by derekstavis** [comment]
Perhaps then we could call it `lightweight-fzf` or `light-fzf`? I also like the mention on the description!

**Comment by patrickf3139** [comment]
@derekstavis thank you for your help and for being one of the last volunteers maintaining omf. 

I've updated the description. I don't like the idea of renaming the whole repo/plugin to lightweight-fzf just for this because it's only lightweight when juxtaposed with Jethro's fzf. And I didn't want to give it a name that would imply it not being full-featured when IMO, it is.

**Comment by patrickf3139** [comment]
Hi Derek, since we last chatted, I've done a more thorough analysis of my `fzf-fish-integration` vs `jethrokuan/fzf`. I actually wrote it out in the readme [here](https://github.com/patrickf3139/fzf-fish-integration#prior-art). The tldr is `fzf-fish-integration` includes the core functionality of `jethrokuan/fzf`. leaves out redundant or buggy functionality, and adds two additional features. I also had a conversation with Jethro offline and he agrees that it'd probably be best if mine overtakes his plugin. As a result, I changed the description of the plugin to be more assertive.

**Comment by patrickf3139** [comment]
Hi @bobthecow, very sorry to @ you when you are probably busy and have nothing to do with this PR. However, I noticed you are one of the last people with write permissions in the oh-my-fish group who is still active. Would you mind taking a quick look and possibly merging, please? Thank you. I've been blocked on this for a long time now :)

**Review by scorphus — CHANGES_REQUESTED** [review]
Please rename the file to remove the extension (see other examples)

**Comment by scorphus** [comment]
I, as a `fzf` user — installed by means of `omf install fzf` — wouldn't want to have the plugin I use suddenly replaced by another, even if they're similar. So I agree with Derek that the two plugins should be set apart.

IMHO, changing a plugin but maintaining its name should not even be considered.

**Comment by patrickf3139** [comment]
Ah, does adding the .fzf to the end not differentiate it to OMF? i.e. would OMF get confused on which one to install?
Either way I am happy to change the name.

**Comment by scorphus** [comment]
Yeah, I could see there were no changes — no red lines and etc. There would be if `.fish` would be stripped from the name, as I thought was the case. Hence the name clash. It simply felt strange *to me* having that in the name. Even more when none of the other packages have an extension — especially `.fish`.

File: packages/fzf.fish [hunk 1]
```diff
@@ -0,0 +1,4 @@
+type = plugin
+repository = https://github.com/patrickf3139/fzf.fish
+maintainer = patrickf3139
+description = Integrate fzf into fish with mnemonic keybindings. A newer, more feature-rich, more maintained alternative `jethrokuan/fzf`.
```
Comment (scorphus) [inline packages/fzf.fish:4]:
```suggestion
description = Integrate fzf into fish with mnemonic keybindings. A newer, more feature-rich, more maintained alternative to `jethrokuan/fzf`.
```

**Review by scorphus — CHANGES_REQUESTED** [review]

**Comment by patrickf3139** [comment]
Hi Pablo, yeah I don't think .fish will get stripped from the name. The .fish is kind of a common extension for fish plugins now! Let me show you:
https://github.com/jorgebucaran/bax.fish
https://github.com/jorgebucaran/awesome.fish
https://github.com/mattgreen/lucid.fish
https://github.com/jorgebucaran/cookbook.fish
https://github.com/patrickf3139/colored_man_pages.fish

Anyway, do you want me to rename the package name still?

**Comment by scorphus** [comment]
Oh, don't get me wrong, sorry for not being clear.

> Anyway, do you want me to rename the package name still?

No, not at all. I was only trying to explain the mere fact that it read strange to me and led me to think that was a mistake and it would end up replacing `fzf`. IOW, I overlooked the PR — my apologies.

**Comment by patrickf3139** [comment]
Glad we're on the same page. Do you mind merging now?

**Comment by patrickf3139** [comment]
Hello again. Mind if I bump?

**Pull request closed without merging**